src/Rules.php

```diff
@@ -99,6 +100,16 @@ public function asArray(): array
         return $arrayOfRules;
     }
 
+    public function isNestedEach(): bool
+    {
+        $array = $this->asArray();
+        if (ArrayHelper::getValue($array, [0, 0]) !== 'each') {
+            return false;
+        }
+
+        return !(ArrayHelper::getValue($array, [0, 1, 0]) === 'nested');
```

Comment:
```suggestion
        return ArrayHelper::getValue($array, [0, 1, 0]) !== 'nested';
```
Why not like this?

Reply:
This was probably suggested by StyleCI at some point. Indeed, can be simplified. Thanks.

Reply:
:heavy_check_mark: